apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -242,6 +243,27 @@
         .dynamic(true)
         .buildWithDefault(Collections.singletonList(WildcardMatcher.valueOf("(?-i)*Nested*Exception")));
 
+    private final ConfigurationOption<List<WildcardMatcher>> ignoreExceptions = ConfigurationOption
+        .builder(new ListValueConverter<>(new WildcardMatcherValueConverter()), List.class)
+        .key("ignore_exceptions")
+        .tags("added[1.11.0]")
+        .configurationCategory(CORE_CATEGORY)
+        .description("A list of exceptions that should be ignored and not reported as errors.\n" +
+            "This allows to ignore exceptions thrown in regular control flow that are not actual errors\n" +
+            "\n" +
+            WildcardMatcher.DOCUMENTATION + "\n" +
+            "\n" +
+            "Examples, all of them are equivalent:\n" +
```

Comment:
They will all cause the `com.mycompany.ExceptionToIgnore` to be ignored, but they are not equivalent

Reply:
Good catch, I forgot to re-generate doc after changing this. I've removed the 'all of them are equivalent'.

Reply:
fixed in next commit.